### PR TITLE
AMP-55921 Added missing 'Event With Template Properties'

### PR DESCRIPTION
**Pull request by falconandy**

### CODE REVIEW

**Review by justin-fiedler — CHANGES_REQUESTED** [review]
Thanks for adding these. Can you please fix the build error
image

**Comment by falconandy** [comment]
@justin-fiedler Fixed - now all tests pass.
For some reason all 'Merge Check' fail
```
Run ampli status --is-merged -t ***
✘ ERROR Authentication failed. Try running ampli login to login. .
```

File: android/java/v1/AmpliApp/ampli.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
   "Branch": "main",
   "Version": "0.0.0",
   "Runtime": "android:java-ampli",
-  "VersionId": "79154a50-f057-4db5-9755-775e4e9f05e6"
+  "VersionId": "79154a50-f057-4db5-9755-775e4e9f05e6",
+  "OmitApiKeys": true
```
Comment (justin-fiedler) [inline android/java/v1/AmpliApp/ampli.json:10]: Thanks for adding omit keys!

**Review by justin-fiedler — APPROVED** [review]
I think merge check is failing due to staging update in progress. Merging as is. Will address merge checks later is problem persists.

**Pull request merged**